### ISSUE TITLE
some questions

### ISSUE THREAD

**Issue by huan2950** [sowmen/face-cutout#8]

**Comment by huan2950** [comment]
the codes in dataset.py where function __getitem__  should be `ori_image = cv2.cvtColor(ori_image, cv2.COLOR_BGR2RGB) `rather than `ori_image = cv2.cvtColor(image, cv2.COLOR_BGR2RGB)` ?

**Comment by sowmen** [comment]
Yes, this was due to a typo made while cleaning up the code. It should be `ori_image = cv2.cvtColor(ori_image, cv2.COLOR_BGR2RGB)`
